shared/src/persistence/dynamo/users/updateUserRecords.js

```diff
@@ -0,0 +1,63 @@
+const client = require('../../dynamodbClientService');
+
+exports.updateUserRecords = async ({
+  applicationContext,
+  oldUser,
+  updatedUser,
+  userId,
+}) => {
+  await client.delete({
+    applicationContext,
+    key: {
+      pk: `section|${oldUser.section}`,
+      sk: `user|${userId}`,
+    },
+  });
+
+  await client.put({
+    Item: {
+      pk: `user|${userId}`,
+      sk: `user|${userId}`,
+      ...updatedUser,
+      userId,
+    },
+    applicationContext,
+  });
+
+  await client.delete({
+    applicationContext,
+    key: {
+      pk: `${oldUser.role}|${oldUser.name}`,
```

Comment:
should this be uppercase?

Reply:
... and the other similar ones in here?

Reply:
I think we've determined this should be ok, yeah?